roles/openshift-emptydir-quota/tasks/main.yaml

```diff
@@ -3,7 +3,7 @@
   replace:
     dest: /etc/origin/node/node-config.yaml
     regexp: '^(.*)perFSGroup: (\s+.*)?$'
-    replace: '\1 perFSGroup: 512Mi\2'
+    replace: '\1 perFSGroup: {{ openshift_node_local_quota_per_fsgroup }}\2'
```

Comment:
@e-minguez in this pull then we will need to include the other providers openshift install vars file